react/week2/todolist/src/ToDoList.js

```diff
@@ -0,0 +1,80 @@
+import React,{ useState } from 'react'
+
+const toDos = [
+    {
+        id: 1,
+        description : "Finish Homework",
+    },
+    {
+        id: 2,
+        description : "Buy Grocceries",
+    },
+    {
+        id : 3,
+        description: "Pickup Kids",
+    },
+    {
+        id : 4,
+        description : "Do the dishes"
+    }
+]
+const ToDoList = ()=> {
+    const [todo,setTodo] = useState([]);
+
+    const addTodo = () =>{
+        const randomTodo = toDos[Math.floor(Math.random()*toDos.length)];
+        const newTodos = todo.concat(randomTodo)
+        setTodo(newTodos);
+    };
+
+    const showList = (props) =>{
+        const{todoitems} = props;
+        const onDelete = (id) =>{
+            const deleteTodo = todo.filter((todos)=>{
+                return todos.id !==id;
+            })
+            .map((e) =>{
+                return e
+            });
+            return setTodo(deleteTodo);
+        };
+    }
+    const [isChecked,setIschecked] = useState(false);
+    const isDone = (id)=>{
+        todo.map((todo)=>{
```

Comment:
should probably use `forEach` here instead of `map`, since you don't use the array returned by map (not a big deal though)